scipy/stats/_binomtest.py

```diff
@@ -313,16 +314,37 @@ def binomtest(k, n, p=0.5, alternative='two-sided'):
             # special case as shortcut, would also be handled by `else` below
             pval = 1.
         elif k < p * n:
-            i = np.arange(np.ceil(p * n), n+1)
-            y = np.sum(binom.pmf(i, n, p) <= d*rerr, axis=0)
+            y = n-binary_search_for_binom_tst(a_fn,d*rerr,np.ceil(p * n),n)+1
```

Comment:
```suggestion
            y = n-binary_search_for_binom_tst(a_fn, d*rerr, np.ceil(p * n), n)+1
```
PEP8 whitespace needed here and below

Reply:
Fixed with flake8 check.